ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala

```diff
@@ -160,5 +171,19 @@ class ClientToolControllerSpec extends PlaySpec with GuiceOneServerPerSuite with
       status(result) mustBe OK
       contentType(result).get mustBe "application/zip"
     }
+
+    "error when when crypt client can't connect to crypt server" in {
+      val application = builder.overrides(bind[WSClient].to(mockClientFailingCrypt)).build
+      val controller = application.injector.instanceOf[ClientToolController]
+
+      try {
+        status(call(controller.downloadClientToolBundle(UUID.randomUUID()), request))
+        throw new Exception("shouldn't happen")
+      } catch {
+        case _: RemoteApiError =>
```

Comment:
Empty right hand side? What does that mean?

Reply:
It means to ignore this particular exception.

Reply:
So here it means: test that controller.downloadClientToolBundle() throws an RemoteApiError.